hack/set-motd/main.go

```diff
@@ -0,0 +1,165 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"log"
+	"os"
+	"time"
+)
+
+type stringFlag struct {
+	set   bool
+	value string
+}
+
+func (sf *stringFlag) Set(x string) error {
+	sf.value = x
+	sf.set = true
+	return nil
```

Comment:
return nil is not needed if only assign values without composition

Reply:
Actually this is my bad, I just rushed developing the script and just assigned values instead of assigning with set. Will change it. :)

Reply:
No change needed, I retested the code and the set value is being checked correctly:
```
falsetruefalsetrue[aranda@hyperion set-motd]$ go run main.go set -path dummy
MOTD true? false
User true? false
PR true? false
Path true? true

[aranda@hyperion set-motd]$ go run main.go set -path dummy -user fbac -pr x 
MOTD true? false
User true? true
PR true? true
Path true? true

[aranda@hyperion set-motd]$ go run main.go set -path dummy -user fbac -pr x -motd x
MOTD true? true
User true? true
PR true? true
Path true? true

[aranda@hyperion set-motd]$ go run main.go set -path dummy -user -pr x -motd x
MOTD true? false
User true? true
PR true? false
Path true? true

[aranda@hyperion set-motd]$ go run main.go set -path dummy -pr x -motd x
MOTD true? true
User true? false
PR true? true
Path true? true
```

Reply:
I'm not talking about that It could works or not, but the code is not good...You don't need to return anything just to assign simple types. Over-coding is something to reduce to keep go simple